starfish/pipeline/features/spots/detector/_base.py

```diff
@@ -1,9 +1,30 @@
+from typing import Union
+
+import xarray as xr
+import numpy as np
+
 from starfish.image import ImageStack
 from starfish.pipeline.algorithmbase import AlgorithmBase
+from starfish.pipeline.features.spot_attributes import SpotAttributes
+from starfish.intensity_table import IntensityTable
 
 
 class SpotFinderAlgorithmBase(AlgorithmBase):
-    # TODO ambrosejcarr: should return IntensityTable when refactor is complete
-    def find(self, hybridization_image: ImageStack):
-        """Find spots."""
+    def find(self, hybridization_image: ImageStack, blobs_image: Union[np.ndarray, xr.DataArray]) \
```

Comment:
formatting -- would prefer:

```
def find(
    self,
    hybridization_image: ImageStack,
    blobs_image: Union[np.ndarray, xr.DataArray]),
) -> IntensityTable:
```

Reply:
Also, I think it would be prudent to consider whether we should accept one of the two (np.ndarray vs xr.DataArray), rather than either.

Reply:
Yea I think when you land your PR we'll make it all xr.DataArray, but right now I have to hedge. 😛